scripts/tests/ec2-memory-benchmark/ec2-memory-benchmark-remote.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/sh
+
+DATAVERSE_APP_DIR=/usr/local/glassfish4/glassfish/domains/domain1/applications/dataverse; export DATAVERSE_APP_DIR
+
+# restart glassfish: 
+
+systemctl restart glassfish
+
+# obtain the pid of the running glassfish process:
+
+GLASSFISH_PID=`ps awux | grep ^glassfi | awk '{print $2}' `; export GLASSFISH_PID 
+
+if [ ${GLASSFISH_PID}"x" = "x" ]
+then
+    echo "Failed to obtain the process id of the running Glassfish process!"
+    exit 1;
+fi
+
+
+# install gnuplot, quietly: 
+
+yum -y install gnuplot >/dev/null 
```

Comment:
For "yum install" to work, the script has to be run as root, right? Should we do a check for this above? Check to make sure the script is being run as the root user, I mean.

Reply:
There are other things in the script that need root. It is run by the parent script as "sudo ec2-memory-benchmark-remote.sh". 
In the current setup, this second (remote) script should never be run by the end user directly.

Reply:
Ah, I didn't notice the sudo. Thanks.